public/app/features/admin/EnterpriseAuthFeaturesCard.tsx

```diff
@@ -0,0 +1,116 @@
+import { css } from '@emotion/css';
+import { useState } from 'react';
+
+import { GrafanaTheme2 } from '@grafana/data';
+import { GrafanaEdition } from '@grafana/data/internal';
+import { t, Trans } from '@grafana/i18n';
+import { config } from '@grafana/runtime';
+import { Text, Box, Icon, Stack, useStyles2, Button, LinkButton } from '@grafana/ui';
+import { contextSrv } from 'app/core/core';
+import { backendSrv } from 'app/core/services/backend_srv';
+
+export interface Props {
+  page?: 'teams' | 'users';
+}
+
+export function EnterpriseAuthFeaturesCard({ page }: Props) {
+  const styles = useStyles2(getStyles);
+  const helpFlags = contextSrv.user.helpFlags1;
+  const HELP_FLAG_ENTERPRISE_AUTH = 0x0004;
+  const [isDismissed, setDismissed] = useState<boolean>(Boolean(helpFlags & HELP_FLAG_ENTERPRISE_AUTH));
+
+  const onDismiss = () => {
+    backendSrv
+      .put(`/api/user/helpflags/${HELP_FLAG_ENTERPRISE_AUTH}`, undefined, { showSuccessAlert: false })
+      .then((res) => {
+        contextSrv.user.helpFlags1 = res.helpFlags1;
+        setDismissed(true);
+      });
+  };
+
+  // This card is only visible in oss
+  if (isDismissed || !isOpenSourceBuildOrUnlicenced()) {
+    return null;
+  }
+
+  return (
+    <Box
+      padding={2}
+      borderColor="weak"
+      borderStyle={'solid'}
+      borderRadius="default"
+      backgroundColor="secondary"
+      display={'flex'}
+      direction="column"
+      gap={1.5}
+      marginTop={3}
+    >
+      <Stack direction="row" alignItems="center" justifyContent={'space-between'}>
+        <div className={styles.cloudBadge}>
```

Comment:
will break this out into reusable component in the next PR that will leverage this design (cloud alerting features info box)